### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by ohler55**
Updated lesson one to use the latest UI and wabur init.

### CODE REVIEW

File: tutorial/lesson-1/README.md [hunk 1]
```diff
@@ -43,15 +44,27 @@ to test the View or UI.
...
 ├── lib
-|   └── ui
-|       └── entry.rb
-└── pages
+|   └── ui_controller.rb
+└── site
+    └── assets
+        └── css
+        └── fonts
+        └── index.html
```
Comment (ashmaroli) [inline tutorial/lesson-1/README.md]: an `index.html` within the `assets` directory?

Comment (ohler55) [reply tutorial/lesson-1/README.md]: oops

File: tutorial/lesson-1/README.md [hunk 2]
```diff
@@ -71,32 +84,60 @@ The View runs in the client or in the normal case, a browser. The WABuR
...
+a project in GitHub and cloning it. Then `cd` to the project
+directory. Assuming the project was called `blog` execute the following
+commands.
 
-[tell me more...](MORE.md#implementation)
+```
+$ mkdir blog
+$ cd blog
+$ wabur init Entry
+```
```
Comment (ashmaroli) [inline tutorial/lesson-1/README.md]: please add reference to `wabur new` as well:
```
Or to start from scratch, locally, use `wabur new` mode. Simply run the following to create a project
directory at './blog' and initialize controllers for type Entry:

    wabur new Entry --base blog

```

**Comment by ashmaroli** [comment]
I think it would be better if the JSON API at various `GET` / `PUT` endpoints for this tutorial app, could be documented as well...

**Comment by ohler55** [comment]
I agree, all the HTTP exchanges should be covered. I figured that would come later when the logging was discussed. Currently planned for lesson 3.

**Comment by ashmaroli** [comment]
okay..

**Comment by ohler55** [comment]
The logging shows the content of the calls so it is easier to explain with the call contents on the screen.

File: tutorial/lesson-1/README.md (hunk 2 above)
Comment (ohler55) [reply tutorial/lesson-1/README.md]: done

**Pull request merged**